src/main/scala/fi/aalto/cs/apluscourses/intellij/utils/ModuleUtils.scala

```diff
@@ -74,6 +81,29 @@ object ModuleUtils {
     }
   }
 
+  @NotNull
+  def getPrettyKeyMapString(@NotNull actionId: String): String = {
+    val shortCuts = KeymapManager
+      .getInstance
+      .getActiveKeymap
+      .getShortcuts(actionId)
+
+    if (shortCuts.nonEmpty) {
+      shortCuts
+        .head
+        .toString
+        .replace("[", "")
+        .replace("]", "")
+        .split(" ")
+        .filter(part => !part.equals("pressed"))
```

Comment:
Nit: could use the underscore syntax here, also, I _think_ `==` works for scala strings, so this could be `.filter(_ != "pressed)`